package-plugin/action.yml

```diff
@@ -0,0 +1,119 @@
+name: "Grafana Package Plugin"
+description: "Packages a Grafana plugin"
+
+outputs:
```

Comment:
you can also output here the plugin id for ease of use.

Reply:
Good call. I did that and removed the `metadata` steps in the example.